api/agent/call.go

```diff
@@ -331,7 +331,7 @@ func (c *call) End(ctx context.Context, err error) {
 	case context.DeadlineExceeded:
 		c.Status = "timeout"
 	default:
-		// XXX (reed): should we append the error to logs? Error field?
+		// XXX (reed): should we append the error to logs? Error field? (TR) think so, otherwise it's lost looks like?
```

Comment:
yea, want to do this https://github.com/fnproject/fn/issues/331 -- could do both, but it seems weird to add things to user logs that may not come from their code. if it were me, i'd go digging through all the code looking for some error string only to not find it and possibly go insane (possibly already insane?)

(want to add migrations to db so we don't bust everyone's db any time we add a field)

Reply:
Perhaps we could prefix the error with something about where it came from.